package.json

```diff
@@ -118,6 +118,7 @@
     "@ledgerhq/hw-transport-node-hid": "6.6.0",
     "@liskhq/lisk-client": "6.0.0-alpha.2",
     "@tanstack/react-query": "4.0.10",
+    "@tanstack/react-query-devtools": "4.2.1",
```

Comment:
do we need it ?

Reply:
@soroushm Since it's only a development tool, it makes sense to have it to debug queries and mutations when necessary